### PR TITLE
Update to latest challenge standard definition

### PR DESCRIPTION
**Pull request by Z-a-r-a-k-i**

### CODE REVIEW

File: front/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM    php:7.3-apache
```
Comment (moul) [inline front/Dockerfile:1]: keep nginx please

Comment (Z-a-r-a-k-i) [reply front/Dockerfile:1]: okay, do you want nginx for all default web challenges in pathwar repo as well ?

Comment (moul) [reply front/Dockerfile:1]: nop, just for this one

other levels require php, this one only hosts static files

Comment (Z-a-r-a-k-i) [reply front/Dockerfile:1]: Understood !

**Pull request closed without merging**